xml/tuning_memory.xml

```diff
@@ -593,13 +593,13 @@ dd if=/dev/zero of=zerofile ibs=1048576 count=$((MEMTOTAL_MBYTES*30/100))
      There are cases when THP may be undesirable. Workloads with sparse memory
      access patterns can perform poorly with THP due to excessive memory
      usage. For example, 2 MB of memory may be used at fault time instead of 4
-     KB for each fault and ultimately lead to premature page reclaim. On
+     KB for each fault and ultimately lead to premature page reclaim. <!-- On
      releases older than <phrase os="sles;sled">&sle; 12 SP2</phrase><phrase
      os="osuse">&opensuse; Leap 42.2</phrase>, it was possible for an
      application to stall for long periods of time trying to allocate a THP
      which frequently led to a recommendation of disabling THP. Such
      recommendations should be re-evaluated for <phrase os="sles;sled">&sle; 12
-     SP3</phrase><phrase os="osuse">&opensuse; Leap 42.3</phrase>
+     SP3</phrase><phrase os="osuse">&opensuse; Leap 42.3</phrase> -->
```

Comment:
This sounds like something that could still be a helpful `<note>`. I would probably not comment it away.